azure/packages/azure-client/.eslintrc.js

```diff
@@ -5,8 +5,32 @@
 
 module.exports = {
 	extends: [require.resolve("@fluidframework/eslint-config-fluid/strict"), "prettier"],
-	plugins: ["eslint-plugin-jsdoc"],
 	parserOptions: {
 		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
 	},
+	rules: {
+		// Useful for developer accessibility
+		"unicorn/prevent-abbreviations": [
+			"error",
+			{
+				allowList: {
+					// Industry-standard index variable name.
+					i: true,
+
+					// Existing export - renaming would be a breaking change.
+					// Leaving this alone for now, but this can be reconsidered in the future.
+					AzureClientProps: true,
```

Comment:
Interested in reviewers' thoughts here. We could deprecate the old name and export the new one, if we really wanted to be strict about this sort of thing. Personally, I'm leaning towards leaving this alone and just applying the rule going forward, but I do think this is something we should take seriously.

Reply:
I also don't feel a need to change this... `Props` is not quite `i` but I imagine with React out there, it might not be too obscure?

Reply:
Fair. I'll update the rules to make this a more general exception and update the docs.